CHANGELOG.md

```diff
@@ -72,6 +72,7 @@ Here is an overview of all new **experimental** features:
 ### Fixes
 
 - TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
+- **General**: Fixed the token-permission and pinned-dependencies issue reported by the scorecard. ([#6085](https://github.com/kedacore/keda/issues/6085))
```

Comment:
Let's improve it to make it more clear.
```suggestion
- **General**: Lock down GitHub token permissions ([#6085](https://github.com/kedacore/keda/issues/6085))
- **General**: Pin GitHub Action actions to specific version ([#6085](https://github.com/kedacore/keda/issues/6085))
```

Also, please move this under improvements as this is not a fix for KEDA itself

Reply:
To the `Other `section please